security/dns.go

```diff
@@ -42,7 +42,8 @@ func dnsLinux() []string {
 }
 
 func dnsMacOS() []string {
-	cmd := exec.Command("networksetup", "-getdnsservers", "Wi-Fi")
+	cmdString := `scutil --dns | grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}'`
+	cmd := exec.Command("sh", "-c", cmdString)
```

Comment:
Interesting approach. Returns duplicate entries on my machine:
```bash
~/go/src/github.com/senorprogrammer/wtf master
❯ scutil --dns | grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}'
1.1.1.1
1.0.0.1
1.1.1.1
1.0.0.1
```

Reply:
Could maybe fix that with a `sort -u`?

Reply:
fixed. it may be not so elegant.

https://superuser.com/questions/734063/what-is-a-scoped-dns-query
about the scoped queried. I don't know what setting will break the code, but it looks like working well for me.